tests/unit/fixtures/auth.json

```diff
@@ -0,0 +1,38 @@
+[
+{
+ "model": "myauth.User",
```

Comment:
Don't think these are necessary if you use the test utils that oscar provides for testing baskets.